utils/ansible_playbook_to_role.py

```diff
@@ -61,7 +61,12 @@ def dict_constructor(loader, node):
 ])
 
 PROFILE_WHITELIST = set([
+    "anssi_nt28_enhanced",
+    "anssi_nt28_high",
+    "anssi_nt28_intermediary",
+    "anssi_nt28_minimal",
```

Comment:
The ANSSI profiles for RHEL8 have a slightly different ID, with `bp28` instead of `nt28`.
And note that High is still in draft state.

Reply:
got it.. will update bp28 and understand that high is in draft.. as mentioned prior, this was just to add some new profiles that may benefit from this in the near future. Currently I plan on supporting the STIG profiles, and will add jobs in our GH workflow to only build and produce roles for RHEL STIG.